.github/workflows/cd-staging.yml

```
@@ -7,6 +7,7 @@ on:
 
 jobs:
   build:
+    if: false
```

actionlint (1.7.10)

constant expression "false" in condition. remove the if: section

(if-cond)